src/main/scala/com/ironcorelabs/DeviceAddResult.scala

```diff
@@ -0,0 +1,41 @@
+package com.ironcorelabs.scala.sdk
+
+import com.ironcorelabs.{sdk => jsdk}
+import java.{util => ju}
+
+/**
+ * Account's device context. Needed to initialize the SDK with a set of device keys.
```

Comment:
This comment seems wrong. It's the AddResult.

Reply:
Good catch

Reply:
Fixed.